html/pfappserver/root/src/views/Configuration/dns/_components/TheForm.vue

```diff
@@ -42,8 +45,21 @@ export const setup = (props) => {
 
   const schema = computed(() => schemaFn(props))
 
+  // props.form come in undefined.
+  // Adding a watcher to trigger when it gets a value fail.
+  // I don't know how to work around it, so this form doesn't work
+
+  // The idea is to clone the form and link it to the UI form.
+  // UI form would use whatever UI components need and backend form would get computed.
+  const uiForm = _.cloneDeep(props.form)
+  uiForm.value['record_dns_in_sql'] = (props.form['record_dns_in_sql'] === "enabled")
+  props.form['record_dns_in_sql'] = computed(() => {
+    return uiForm['record_dns_in_sql'] ? "enabled" : "disabled"
+  })
```

Comment:
This binding could be extracted into a composable, it doesn't need to be part of the form